src/main/scala/it/pagopa/interop/catalogprocess/api/impl/ProcessApiServiceImpl.scala

```diff
@@ -71,13 +71,13 @@ final case class ProcessApiServiceImpl(
       s"Creating E-Service for producer ${eServiceSeed.producerId} with service name ${eServiceSeed.name}"
     logger.info(operationLabel)
     val clientSeed: CatalogManagementDependency.EServiceSeed = Converter.convertToClientEServiceSeed(eServiceSeed)
-    val result                                               = for {
+    val result: Future[OldEService]                          = for {
```

Comment:
Do we still have the dependency on OldEService?

Reply:
+1

Reply:
yes, for POST/PUT endpoints because they have not been yet migrated